Homework2/classes/Bow.ts

```diff
@@ -0,0 +1,17 @@
+// your code goes here
+
+import {Weapon} from "./Weapon";
+
+export class Bow extends Weapon{
+    //constructor
+    constructor(baseDamage: number, baseDurability: number, value: number, weight: number) {
+        super("bow", baseDamage, baseDurability, value, weight);
+    }
+
+    polish(): void {
+        if (this.effectiveDurability < 1) {
+            this.durabilityModifier += Bow.MODIFIER_CHANGE_RATE;
+            this.setDurability();
```

Comment:
Imagine that `this.effectiveDurability` is 0. Calling `bow.polish()` will immediately set its `effectiveDurability` to the value that is bigger than it was during the initialization. However, it should just increase its value by modifier_change_rate.